### ISSUE TITLE
[App]: Seeding a new sweep with previous runs has an implicit limit of 1000 previous runs

### ISSUE THREAD

**Issue by RylanSchaeffer** [wandb/wandb#7372]
### Current Behavior

I will select a number of runs (e.g., 3420 runs from two previous sweeps) to seed a new sweep, but W&B will select a subset (specifically, 1000 runs) for the new sweep. This is new behavior; W&B didn't use to behave as such.

### Expected Behavior

I expect that when I seed a new sweep with > 1000 runs, the sweep will be seeded with the correct number of runs and will not truncate to 1000.

### Screenshots

![image](https://github.com/wandb/wandb/assets/8942987/78b3caec-7f9f-40fa-9ff5-0c26995c68ee)

### Environment

OS: Ubuntu 20.04.4 LTS

Browsers:

Version: wandb, version 0.16.5

**Comment by ArtsiomWB** [comment]
Hi Rylan, apologies you are seeing this behavior on your side! Could you please point me to a workspace where you are experiencing this behavior?

**Comment by RylanSchaeffer** [comment]
I would prefer to not publicly post links :grimacing: 

Is there some way I can privately communicate such information to you?

**Comment by RylanSchaeffer** [comment]
Ehh I care more about having a quick solution.

Here is a sweep with 1140 runs: https://wandb.ai/rylan/signal_prop/sweeps/ejkla2x2

I want to use all 1140 runs to seed a new sweep that will also include a second value of one hyperparameter (the previous sweep has mu=1.0 only, now I want to have a sweep of mu=1.0 and mu=0.0):

When I initialize a new sweep, it has only 1000 previous runs: https://wandb.ai/rylan/signal_prop/sweeps/n5nhqi8e/table?nw=nwuserrylan

**Comment by ArtsiomWB** [comment]
Hi @RylanSchaeffer, when you start a new sweep up from a previous sweep, do you do it using the `create a sweep` button from the runs table tab? I will try to repro this behavior on my side.

**Comment by RylanSchaeffer** [comment]
That's correct. I filter for runs from the previous sweep(s) and then use the Create a Sweep button. I've done this for over a year now and it used to work as intended. This 1000 previous runs cap is new, as best as I can tell.

**Comment by ArtsiomWB** [comment]
When you initialize the sweep, how many runs do you see next to the `configure prior runs` button?

**Comment by ArtsiomWB** [comment]
Hi there, I wanted to follow up on this request. Please let us know if we can be of further assistance or if your issue has been resolved.

**Comment by RylanSchaeffer** [comment]
Here I am selecting 1140 runs:

![image](https://github.com/wandb/wandb/assets/8942987/92a26d0d-c095-43f9-82b5-a937386ca8f5)

I then press "Create Sweep". I specify additional values for a hyperparameter `mu` such that the finished sweep will be a strict subset of the new sweep:

![image](https://github.com/wandb/wandb/assets/8942987/8c12bc5e-c506-4005-a98e-ae8c5fb87aa2)

I then press "Initialize Sweep". If I then navigate to the new sweep page, there are 1000 prior runs, even though I selected 1140 runs.

![image](https://github.com/wandb/wandb/assets/8942987/b2fe8881-33a7-4c7d-adf6-b3395aaedb16)

I hope this helps!

**Comment by ArtsiomWB** [comment]
Thank you so much for sending over all of the screenshots, this helps a ton. I checked internally and this is a known bug. I have went ahead and re-escalated the issue and our engineering and product teams will look into it. 

Apologies for the inconvenience. If we have any follow up questions regarding this behavior, we will put them in here.

**Comment by RylanSchaeffer** [comment]
Awesome thank you!

**Comment by ArtsiomWB** [comment]
Totally will! By the way, How long ago were you able to create sweeps from over 1k runs? I do see that the behavior you are currently experiencing is a long-standing bug that was marked a back logged because our engineering team could not reproduce it before.

**Comment by ArtsiomWB** [comment]
Hey @RylanSchaeffer, are you still experiencing this issue on your end?

**Comment by RylanSchaeffer** [comment]
I'm no longer using the feature :(

I realized I couldn't risk spending a bunch of compute on a sweep that couldn't be used to seed a subsequent sweep if necessary. Now I structure my sweeps differently.

But thank you for checking!

**Comment by ArtsiomWB** [comment]
Ah, thank you for the update, @RylanSchaeffer!  I'll go ahead and close this thread out for tracking purposes then. Please feel free to write in whenever you have any more questions
